tasks/uncss.js

```diff
@@ -34,17 +34,17 @@ module.exports = function ( grunt ) {
                 }
             });
 
-            if ( src.length === 0 && file.src.length === 0 ) {
-                grunt.fail.warn( 'Destination (' + file.dest + ') not written because src files were empty.' );
-            }
-
-            file.src.forEach(function (source) {
+            file.orig.src.forEach(function (source) {
```

Comment:
Getting `file.orig.src.forEach is not a function` here, fix in my PR against your repo https://github.com/kfiku/grunt-uncss/pull/1